File: app/src/main/java/github/leavesc/databindingsamples/BlankFragment.java
package github.leavesc.databindingsamples;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.databinding.DataBindingUtil;
import androidx.fragment.app.Fragment;
import github.leavesc.databindingsamples.databinding.FragmentBlankBinding;

/**
 * 作者：CZY
 * 时间：2020/6/29 17:01
 * 描述：
 */
public class BlankFragment extends Fragment {

    public BlankFragment() {

    }

    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        FragmentBlankBinding binding = DataBindingUtil.inflate(inflater, R.layout.fragment_blank, container, false);
        binding.setHint("Hello");
        return binding.getRoot();
    }

}

File: app/src/main/java/github/leavesc/databinding/BlankFragment.kt
package github.leavesc.databinding

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.databinding.DataBindingUtil
import androidx.fragment.app.Fragment
import github.leavesc.databinding.databinding.FragmentBlankBinding

/**
 * 作者：leavesC
 * 时间：2020/6/29 23:02
 * 描述：
 * GitHub：https://github.com/leavesC
 */
class BlankFragment : Fragment() {

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        val binding: FragmentBlankBinding =
            DataBindingUtil.inflate(inflater, R.layout.fragment_blank, container, false)
        binding.hint = "Hello"
        return binding.root
    }

}
